packages/block-library/src/query-title/block.json

```diff
@@ -3,10 +3,18 @@
 	"name": "core/query-title",
 	"title": "Query Title",
 	"category": "design",
-	"description": "Display the query title.",
+	"description": "Display the query title based on the queried object. Supports search, archive and 404 templates. Dynamic content is available on the search template for: %search%, %total%.",
 	"textdomain": "default",
+	"usesContext": [ "templateSlug" ],
 	"attributes": {
-		"type": {
+		"content": {
+			"type": "string",
+			"default": "Query title placeholder"
```

Comment:
I'd avoid non-translatable default values in `block.json`.

Put the default text in the edit component instead and make it translatable.